### ISSUE TITLE
fill-opacity is only applied to one element, subsequent elements ignored

### ISSUE THREAD

**Issue by yGuy** [yWorks/jsPDF#5]
This has been originally reported in https://github.com/yWorks/svg2pdf.js/issues/29 -there is also a proposed solution.

**Comment by TorsteinHonsi** [comment]
Thanks!

**Comment by TorsteinHonsi** [comment]
I tried this change locally and it works fine:
```js
      activeFontKey = fontState.key;
      activeFontSize = fontState.size;

      activeGState = null;

      return this;
    };
```
Will you do the change in the repo?

**Comment by yGuy** [comment]
Should be fixed with the newest 1.2.69 (along with the broken build).

**Comment by TorsteinHonsi** [comment]
Great, thanks! 

FYI, the version number in the header seems to be forgotten: https://github.com/yWorks/jsPDF/blob/v1.2.69/dist/jspdf.min.js#L3
